packages/flame/lib/src/components/mixins/collidable.dart

```diff
@@ -19,15 +19,6 @@ mixin Collidable on Hitbox {
 
   void onCollision(Set<Vector2> intersectionPoints, Collidable other) {}
   void onCollisionEnd(Collidable other) {}
-
-  @override
-  void onRemove() {
-    final parentGame = findParent<FlameGame>();
-    if (parentGame is HasCollidables) {
-      parentGame.collidables.remove(this);
```

Comment:
So how is the Collidable removed from the list now?

Reply:
I removed this list entirely